templates/miq-template-ext-db.yaml

```diff
@@ -305,28 +307,7 @@ objects:
               cpu: "${MEMCACHED_CPU_REQ}"
             limits:
               memory: "${MEMCACHED_MEM_LIMIT}"
-- apiVersion: v1
-  kind: Service
-  metadata:
-    name: "${DATABASE_SERVICE_NAME}"
-    annotations:
-      description: Remote database service
-  spec:
-    ports:
-    - name: postgresql
-      port: 5432
-      targetPort: "${{DATABASE_PORT}}"
-    selector: {}
-- apiVersion: v1
-  kind: Endpoints
-  metadata:
-    name: "${DATABASE_SERVICE_NAME}"
-  subsets:
-  - addresses:
-    - ip: "${DATABASE_IP}"
-    ports:
-    - port: "${{DATABASE_PORT}}"
-      name: postgresql
```

Comment:
Are we sure this isn't needed? To be honest I can't remember why we needed the service and endpoint combination rather than using the IP directly in the first place.

Maybe we were originally using the service env vars internally rather than passing in the database URL?
@bdunne do you remember?

Reply:
I've deployed our setup without the service and endpoint so I can assure you it's working fine.
Maybe you did it like this because you are using a service in the other templates and just stuck with it for this template?

As I mentioned in the PR message: endpoints can only be created with an IP address. So if someone (like in my case) uses some DBaaS which requires a FQDN as target server rather than an IP, an endpoint can not be used.

Reply:
Actually I'm surprised this line in the startup for the container doesn't fail https://github.com/ManageIQ/manageiq-pods/blob/master/images/miq-app-frontend/container-assets/check-dependent-services.sh#L8

Reply:
If you removed the service called "postgresql" the environment variably `POSTGRESQL_SERVICE_HOST` would no longer be present on the containers in the project.

Reply:
Good catch. But the failed check does not prevent the MIQ pod to start normally. There is an error message but the startup proceeds:

```
== Checking <IP>:11211 status ==
<IP>:11211 - accepting connections
Error something seems wrong, we need at least two parameters to check service status
== Writing encryption key ==
== Restoring PV data symlinks =
[...]
```